dataflow/src/domain/mod.rs

```diff
@@ -1148,7 +1148,7 @@ impl Domain {
                                     use itertools::Itertools;
 
                                     let mut chunked_replay_tx =
-                                        TcpSender::connect(&domain_addr, Some(1)).unwrap();
+                                        TcpSender::connect(&domain_addr).unwrap();
```

Comment:
So I guess this means we're assuming this being bounded doesn't matter.

Reply:
Which is a *little* questionable, because your chunked replay is going to be sent all at once. Though maybe that's fine. You'll fill up the TCP buffers given that the reader reads round-robin.